requirements/minimal.txt

```diff
@@ -6,4 +6,5 @@ mycroft-messagebus-client~=0.9.1,!=0.9.2,!=0.9.3
 psutil~=5.6.6
 combo-lock~=0.2
 requests-futures~=0.9.5
-ovos-plugin-manager>=0.0.2
\ No newline at end of file
+ovos_utils~=0.0.14a1
```

Comment:
Can this wait until a stable version spec? Or a comment to make sure it's bumped before an ovos-core non-alpha release

Reply:
a PR was merged that already depends on this version, so better to have an alpha than broken dev branch